File: CSharp/OpenTraceability/Models/Common/StandardBusinessDocumentHeader.cs
using OpenTraceability.Utility.Attributes;
using System;

namespace OpenTraceability.Models.Common
{
    public class StandardBusinessDocumentHeader
    {
        [OpenTraceability(Constants.SBDH_NAMESPACE, "HeaderVersion")]
        public string HeaderVersion { get; set; } = string.Empty;

        [OpenTraceabilityObject]
        [OpenTraceability(Constants.SBDH_NAMESPACE, "Sender")]
        public SBDHOrganization Sender { get; set; }

        [OpenTraceabilityObject]
        [OpenTraceability(Constants.SBDH_NAMESPACE, "Receiver")]
        public SBDHOrganization Receiver { get; set; }

        [OpenTraceabilityObject]
        [OpenTraceability(Constants.SBDH_NAMESPACE, "DocumentIdentification")]
        public SBDHDocumentIdentification DocumentIdentification { get; set; }

        public static StandardBusinessDocumentHeader DummyHeader
        {
            get
            {
                var dh = new StandardBusinessDocumentHeader();
                dh.HeaderVersion = "x";
                dh.Sender = new SBDHOrganization()
                {
                    Identifier = "x",
                    ContactInformation = new SBDHContact()
                    {
                        ContactName = "x",
                        EmailAddress = "x"
                    }
                };
                dh.Receiver = new SBDHOrganization()
                {
                    Identifier = "x",
                    ContactInformation = new SBDHContact()
                    {
                        ContactName = "x",
                        EmailAddress = "x"
                    }
                };
                dh.DocumentIdentification = new SBDHDocumentIdentification()
                {
                    CreationDateAndTime = DateTime.UtcNow,
                    InstanceIdentifier = "x",
                    MultipleType = "false",
                    Standard = "x",
                    Type = "x",
                    TypeVersion = "3.0"
                };
                return dh;
            }
        }
    }
}

File: JDK11/opentraceability/src/main/java/opentraceability/models/common/standardbusinessdocumentheader.java
package opentraceability.models.common;

import opentraceability.Constants;
import opentraceability.utility.attributes.OpenTraceabilityAttribute;
import opentraceability.utility.attributes.OpenTraceabilityObjectAttribute;
import java.time.OffsetDateTime;

public class StandardBusinessDocumentHeader {

    @OpenTraceabilityAttribute(Constants.SBDH_NAMESPACE, "HeaderVersion")
    public String HeaderVersion = "";

    @OpenTraceabilityObjectAttribute
    @OpenTraceabilityAttribute(Constants.SBDH_NAMESPACE, "Sender")
    public SBDHOrganization Sender;

    @OpenTraceabilityObjectAttribute
    @OpenTraceabilityAttribute(Constants.SBDH_NAMESPACE, "Receiver")
    public SBDHOrganization Receiver;

    @OpenTraceabilityObjectAttribute
    @OpenTraceabilityAttribute(Constants.SBDH_NAMESPACE, "DocumentIdentification")
    public SBDHDocumentIdentification DocumentIdentification;

    public static StandardBusinessDocumentHeader DummyHeader = new StandardBusinessDocumentHeader();
    static {
        DummyHeader.HeaderVersion = "x";

        DummyHeader.Sender = new SBDHOrganization();
        DummyHeader.Sender.Identifier = "x";
        DummyHeader.Sender.ContactInformation = new SBDHContact();
        DummyHeader.Sender.ContactInformation.ContactName = "x";
        DummyHeader.Sender.ContactInformation.EmailAddress = "x";

        DummyHeader.Receiver = new SBDHOrganization();
        DummyHeader.Receiver.Identifier = "x";
        DummyHeader.Receiver.ContactInformation = new SBDHContact();
        DummyHeader.Receiver.ContactInformation.ContactName = "x";
        DummyHeader.Receiver.ContactInformation.EmailAddress = "x";

        DummyHeader.DocumentIdentification = new SBDHDocumentIdentification();
        DummyHeader.DocumentIdentification.CreationDateAndTime = OffsetDateTime.now();
        DummyHeader.DocumentIdentification.InstanceIdentifier = "x";
        DummyHeader.DocumentIdentification.MultipleType = "false";
        DummyHeader.DocumentIdentification.Standard = "x";
        DummyHeader.DocumentIdentification.Type = "x";
        DummyHeader.DocumentIdentification.TypeVersion = "3.0";
    }
}